pkg/azuredisk/azuredisk_v2.go

```diff
@@ -181,6 +186,15 @@ func (d *DriverV2) Run(endpoint, kubeConfigPath string, testBool bool) {
 	})
 
 	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	c := make(chan os.Signal, 2)
+	signal.Notify(c, []os.Signal{os.Interrupt, syscall.SIGTERM}...)
```

Comment:
np: Any reason you create a slice here and spread it `...` instead of just passing the two params? Also, for consistency you could pass `syscall.SIGINT` instead of `os.Interrupt` as they are equivalent.